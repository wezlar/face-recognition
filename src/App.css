.App {
  text-align: center;
}

.App-header {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: #333;
}

.App-link {
  color: red;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
