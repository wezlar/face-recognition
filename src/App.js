import React from 'react';
import './App.css';

import Camera from './components/camera/Camera';

function App() {
  return (
    <div className='App'>
      <header className='App-header'>
        <p>
          Test app
        </p>
        <Camera />
      </header>
    </div>
  );
}

export default App;
