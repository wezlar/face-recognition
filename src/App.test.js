import React from 'react';
import {
  render,
  cleanup,
} from '@testing-library/react';

import App from './App';

afterEach(cleanup);

describe('App.js', () => {
  test('to be replaced', () => {
    const { container } = render(<App />);

    expect(container.firstChild).toMatchSnapshot();
  });
});


