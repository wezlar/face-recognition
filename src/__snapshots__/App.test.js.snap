// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.js to be replaced 1`] = `
<div
  class="App"
>
  <header
    class="App-header"
  >
    <p>
      Test app
    </p>
  </header>
</div>
`;
